pkg/tcpip/stack/stack.go

```diff
@@ -46,6 +48,41 @@ const (
 	resolutionAttempts = 3
 )
 
+const (
+	// fakeNetNumber is used as a protocol number in tests.
+	// This constant should match fakeNetNumber in stack_test.go
+	fakeNetNumber    tcpip.NetworkProtocolNumber = math.MaxUint32
+)
+
+// Packet forwarding flags. Forwarding settings for different network protocols
+// are stored as bit flags in an uint32 number.
+const (
+	forwardingIPv4 uint32 = 1 << iota
+	forwardingIPv6
+
+	// forwardingFake is used to test package forwarding with a fake protocol.
+	forwardingFake
+)
+
+func getForwardingFlag(protocol tcpip.NetworkProtocolNumber) uint32 {
+	var flag uint32
+	switch protocol {
+	case header.IPv4ProtocolNumber:
+		flag = forwardingIPv4
+
+	case header.IPv6ProtocolNumber:
+		// IP forwarding in master branch is always off. There is no way to change ipv6 forwarding.
```

Comment:
We have other users of netstack besides runsc. I am not sure that this is a useful comment.

Reply:
fixed